### PR TITLE
Non-functional changes only

### PR DESCRIPTION
**Pull request by bratseth**
@gjoranv please review

Just minor cleanup resulting from reading some code.

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java [hunk 1]
```diff
@@ -322,14 +322,14 @@ public String toString() {
...
-            throw new IllegalStateException("This hit must have a 'uri' field, and this fild must be filled through " +
+            throw new IllegalStateException("This hit must have a 'uri' field, and this field must be filled through " +
                                             "Execution.fill(Result)) before hashCode() is accessed.");
         } else {
             return super.hashCode();
         }
     }
 
-    private static StringBuilder asHexString(StringBuilder sb, GlobalId gid) {
+    private void appendAsHex(GlobalId gid, StringBuilder sb) {
```
Comment (gjoranv) [inline container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: This can and should still be static, right?

**Review by gjoranv — CHANGES_REQUESTED** [review]

File: container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java (hunk 1 above)
Comment (bratseth) [reply container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: Can: Yes, Should: Not imho

Comment (gjoranv) [reply container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: When should a helper that could be static not be declared as such?

Comment (bratseth) [reply container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: When it's not meant to be used in a static context.

Comment (gjoranv) [reply container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: Do as you want. A good SO thread: https://stackoverflow.com/questions/538870/should-private-helper-methods-be-static-if-they-can-be-static

**Review by gjoranv — APPROVED** [review]

**Pull request merged**